modules/tinymce/src/plugins/table/test/ts/browser/settings/TableGridFalseTest.ts

```diff
@@ -18,7 +18,8 @@ describe('browser.tinymce.plugins.table.TableGridFalse', () => {
     await Waiter.pTryUntil('click table menu', () =>
       TinyUiActions.clickOnUi(editor, 'div.tox-menu div.tox-collection__item .tox-collection__item-label:contains("Table")')
     );
-    const dialog = await TinyUiActions.pWaitForDialog(editor, 'div.tox-dialog:has(div.tox-dialog__title:contains("Table Properties"))');
+    const dialog = await TinyUiActions.pWaitForDialog(editor);
```

Comment:
This test fails on Chrome v105 as it introduces `:has` selector support and this doesn't work (I'm assuming since `:contains` is a Sizzle only selector). So while this isn't exactly the same, it does still ensure we're getting the correct dialog.